includes/utilities.inc

```diff
@@ -175,7 +175,11 @@ function islandora_cwrc_document_get_rng_css_accessibility($object) {
 function islandora_cwrc_document_check_xml($file) {
   // Check that the uploaded file is well-formed XML.
   libxml_use_internal_errors(TRUE);
-  simplexml_load_file(drupal_realpath($file->uri));
+  $file_contents = file_get_contents(drupal_realpath($file->uri));
```

Comment:
Looks like it was doing it initially, but this doesn't have to `drupal_realpath()`... Most PHP functions should be able to handle stream wrappers.